tutorials/llama_index_knowledge_base_tutorial.ipynb

```diff
@@ -0,0 +1,402 @@
+{
```

Comment:
<p>This is kinda a leap - how would I get this data? Explain how you would run this llama_index application and log the outputs to files. Without that explanation, this is a pretty big leap from the above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/734/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>